utilities/counted_fs.cc

```diff
@@ -242,6 +244,14 @@ class CountedDirectory : public FSDirectoryWrapper {
     }
     return rv;
   }
+
+  ~CountedDirectory() {
+    if (!closed_) {
+      // TODO: fix DB+CF code to use explicit Close, not rely on destructor
```

Comment:
If the ref counter is feasible, we may already fix this TODO and we can remove these?